locales/en/data-classes.ftl

```diff
@@ -64,6 +65,7 @@ ip-addresses = IP addresses
 job-titles = Job titles
 mac-addresses = MAC addresses
 marital-statuses = Marital statuses
+mnemonic-phrases = Mnemonic phrases
```

Comment:
This is impossible to understand for most people. Let's add a comment

```
# Mnemonic phrases are a group of words used to access the content of cryptocurrency wallets.
```

Reply:
Awesome, thanks. I can do that tomorrow when I’m at my laptop, but are we ok to merge after that, or would you like me to do the other “en-*” locales while I’m at it?